packages/core/lib/use-default-render.tsx

```diff
@@ -0,0 +1,11 @@
+import { ReactNode, useMemo } from "react";
+
+export const useDefaultRender = () => {
+  return useMemo<React.FunctionComponent<{ children?: ReactNode }>>(() => {
+    const PuckDefault = ({ children }: { children?: ReactNode }) => (
+      <>{children}</>
+    );
+
+    return PuckDefault;
+  }, []);
+};
```

Comment:
I can't remember why I used a memo here and didn't just move this to the global scope... possibly just lazy, but possibly some reason.

It might be easier to just have this as a component

```ts
import { ReactNode, useMemo } from "react";

export const DefaultRender = () => {
  const PuckDefault = ({ children }: { children?: ReactNode }) => (
    <>{children}</>
  );

  return PuckDefault;
};
 ```

Reply:
That should be moved to the component directory?

Reply:
That's fine, but maybe it needs a better name... `DefaultOverride`?

Reply:
I've moved to a component and tested custom ui and couldn't see any problems.